src/main/flight/pid.c

```diff
@@ -510,6 +531,31 @@ void pidInitConfig(const pidProfile_t *pidProfile)
     acLimit = (float)pidProfile->abs_control_limit;
     acErrorLimit = (float)pidProfile->abs_control_error_limit;
 #endif
+
+#ifdef USE_DYN_LPF
+    if (pidProfile->dyn_lpf_dterm_idle > 0 && pidProfile->dyn_lpf_dterm_max_hz > 0) {
+        if (pidProfile->dterm_lowpass_hz > 0 ) {
+            dynLpfMin = pidProfile->dterm_lowpass_hz;
+            switch (pidProfile->dterm_filter_type) {
+                case FILTER_PT1:
+                    dynLpfFilter = DYN_LPF_PT1;
+                    break;
+                case FILTER_BIQUAD:
+                    dynLpfFilter = DYN_LPF_BIQUAD;
+                    break;
+                default:
+                    dynLpfFilter = DYN_LPF_NONE;
+                    break;
+            }
+        }
+    } else {
+        dynLpfFilter = DYN_LPF_NONE;
+    }
+    dynLpfIdle = pidProfile->dyn_lpf_dterm_idle / 100.0f;
+    dynLpfIdlePoint = (dynLpfIdle - (dynLpfIdle * dynLpfIdle * dynLpfIdle) / 3.0f) * 1.5f;
+    dynLpfInvIdlePoint = 1 / (1 - dynLpfIdlePoint);
+    dynLpfDiff = pidProfile->dyn_lpf_dterm_max_hz - dynLpfMin;
```

Comment:
you do not need dynLpfDiff global because its used only in one place, rework dynLpfIdlePoint to dynLpfInvIdlePointScaled and use it later, it wil save some cpu time too
`dynLpfInvIdlePointScaled = 1 / (1 - dynLpfIdlePoint) * (pidProfile->dyn_lpf_dterm_max_hz - dynLpfMin);`